README.md

```diff
@@ -59,9 +59,19 @@ wikibaseintegrator~=0.11.3
 # WikibaseIntegrator / WikidataIntegrator #
 
 WikibaseIntegrator (wbi) is a fork of [WikidataIntegrator](https://github.com/SuLab/WikidataIntegrator) (wdi) whose
-purpose is to focus on compatibility with Wikibase. There have been many improvements which have led to radical changes
-in the code. See the [release notes](https://github.com/LeMyst/WikibaseIntegrator/releases) to find out what has
-changed.
+purpose is to focus on an improved compatibility with Wikibase and adding missing functionalities.
+The main differences between these two libraries are :
+
+* A complete rewrite of the library with an object-oriented architecture allowing for easy interaction, data validation
+  and extended functionality
+* Add support for reading and writing Lexeme, MediaInfo and Property datatypes
+* Add support for [Structured Data on Commons](https://commons.wikimedia.org/wiki/Commons:Structured_data) (SDC)
```

Comment:
Isn't SDC just another Wikibase instance? How does WBI support SDC specifically?

Reply:
SDC makes some changes between "claims" and "statements" fields when you compare with a default Wikibase.

Reply:
Interesting! Are these differences documented anywhere?

Reply:
You can found more informations and history in this phabricator task:
https://phabricator.wikimedia.org/T149410

But during my search of this information, I found that Andra added SDC support to WDI, with the sdc_core.py